src/stirling/source_connectors/perf_profiler/perf_profiler_bpf_test.cc

```diff
@@ -453,7 +453,7 @@ TEST_F(PerfProfileBPFTest, PerfProfilerCppTest) {
 }
 
 // TODO(jps/oazizi): This test is flaky.
```

Comment:
Sounds like we can remove this comment as well as part of re-enabling

Reply:
Thank you!